pkg/virt-controller/watch/vm/vm.go

```diff
@@ -2999,16 +2999,11 @@ func (c *Controller) sync(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineI
 	}
 
 	// FIXME(lyarwood): Move alongside netSynchronizer
-	syncedVM, err := c.instancetypeController.Sync(vm, vmi)
-	if err != nil {
+	if syncedVM, err := c.instancetypeController.Sync(vm, vmi); err == nil {
```

Comment:
nit - I'd rather not inline the `err` check here and just handle `err != nil` first and `vm = syncedVM` outside of any conditional. Just makes it easier to read IMHO.